src/cljs/rems/actions/action.cljs

```diff
@@ -0,0 +1,36 @@
+(ns rems.actions.action
+  (:require [re-frame.core :as rf]
+            [rems.atoms :refer [textarea]]
+            [rems.text :refer [text]]))
+
+(defn- action-collapse-id [action-id]
+  (str "actions-" action-id))
+
+(defn button-wrapper [{:keys [id text class on-click]}]
+  [:button.btn.mr-3
+   {:id id
+    :name id
+    :class (or class :btn-secondary)
+    :on-click on-click}
+   text])
+
+(defn cancel-action-button [id]
+  [:button.btn.btn-secondary
+   {:id (str "cancel-" id) :data-toggle "collapse" :data-target (str "#" (action-collapse-id id))}
+   (text :t.actions/cancel)])
+
+(defn- action-comment [label-title comment on-comment]
+  [:div.form-group
+   [:label {:for "judge-comment"} label-title]
+   [textarea {:id "judge-comment"
+              :name "judge-comment" :placeholder (text :t.actions/comment-placeholder)
+              :value comment
+              :on-change on-comment}]])
+
+(defn action-form-view [id title comment-title button content comment on-comment]
```

Comment:
this could use a docstring

Reply:
Yes definitely all the common stuff needs to be documented. 

Probably this can be refactored to be simpler with less params or keyword params.

Reply:
+1 for keyword params when there are over 3 params.

Reply:
Done in latest.